target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -81,7 +81,8 @@ tplink,tl-wr842n-v5)
 	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x1e"
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
 	;;
-tplink,tl-wr840n-v4)
+tplink,tl-wr840n-v4|\
+tplink,tl-wr850n-v2)
 	ucidef_set_led_wlan "wlan2g" "wlan2g" "$boardname:green:wlan" "phy0tpt"
```

Comment:
Please use a DT-trigger instead of this line:
`linux,default-trigger = "phy0tpt";`

Reply:
made changes

Reply:
No, please.

First, this also changes the config for 840n-v4, which should stay unaffected.

What I asked you is to implement the WLAN LED by adding the trigger in the DTS file. You can grep for the line I've provided in the dts directory, and have a look how this is done for other devices. The ucidef_set_led_wlan line will then just be dropped for _your_ device.

Reply:
Now have I done everything right?

Reply:
You've done the right regarding the WiFi LED, but now you miss setup for lan/wan LEDs.

Reply:
You're right
I fixed it